modules/op-specifying-pvc-as-volumesource-in-workspaces.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="specifying-pvc-as-volumesource-in-workspaces_{context}"]
+= Specifying PersistentVolumeClaims as VolumeSource in Workspaces
+
+Workspaces help share data among Tasks and allow you to specify one or more volumes that each Task in the Pipeline requires during execution.
+
+In this section, you will create a PersistentVolumeClaim to provide data storage and bind it to the workspace. This PersistentVolumeClaim provides the volumes or filesystem required for the pipeline execution.
+
+[discrete]
+.Procedure
+
+. Copy the contents of the following sample PersistentVolumeClaim YAML file and save it:
```

Comment:
Suggest: "Copy and save the contents..."
Alternately, you might not want to instruct the user to copy this if it is sample data. So you might say something like, "Create a PersistentVolumeClaim YAML file similar to the following example:"

Reply:
For this entire workflow to work they need to use this particular YAML file hence instructions to copy. Otherwise I like the second option too.